test/controllers/open_library_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class OpenLibraryControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get open_library_index_url
+    assert_response :success
+  end
```

Comment:
What about the "bad_request" scenario? Is this communicating with the open api? You should mock it and test success and failure scenarios

Reply:
this file should be deleted, and use rspec

Reply:
ok